src/olympia/versions/models.py

```diff
@@ -583,10 +594,7 @@ def inherit_nomination(sender, instance, **kw):
     if (instance.nomination is None and
             addon.status in amo.UNREVIEWED_ADDON_STATUSES and not
```

Comment:
I looked into it, and the call from `addons.models.watch_status`, but taking it out only created test fails.  The problem is, with updates, we don't want the version to inherit the nomination date of the original approval, or previous approved updates.  I can't see how it ever worked with updates tbh.

If anything I'm tempted to remove the `inherit_nomination` calls from `post_save` and `watch_status` and only have it set in `handle_upload`